src/reference/interactive/index.pug

```diff
@@ -17,47 +17,49 @@ block reference
     br
     .alert.alert-info
         p.
-            Looking for Interactive 1 docs? Find them #[a(href='/reference/interactive1/index.html') here].
+            For docs about Interactive 1, go to #[a(href='/reference/interactive1/index.html')Interactive 1 docs].
 
     h2#introduction Introduction
 
-    p Beam Interactive is a new way to let viewers directly control the environment in and around streamer’s broadcasts by letting viewers interact using a user interface. When a broadcast has interactivity enabled visual controls will appear beneath the broadcast's video that respond directly to the events of the broadcast. These controls can dynamically update as the broadcast progresses in response to different situations that occur.
+    p This documentation provides an overview of Beam Interactive features and explains how developers can use them to add interactivity into their game or application.
```

Comment:
this is to set expectation what readers are getting when they complete going through the content on this page